components/cards/TransactionSummaryView.tsx

```diff
@@ -59,8 +59,8 @@ function TransactionSummaryView({
 
   const stream =
```

Comment:
@mitesh-mutha Perhaps keep the `stream` field as seconds and update the `streamDisplay` field to convert to annual?

Reply:
See `streamBuffer` below that requires the `stream` in seconds for the correct calculation